src/Site.js

```diff
@@ -385,6 +412,193 @@ Site.prototype.createPage = function (config) {
   });
 };
 
+/**
+ * Converts an existing Github wiki or docs folder to a MarkBind website.
+ */
+Site.prototype.convert = function (rootPath) {
+  return new Promise((resolve, reject) => {
+    this.addIndexPage(rootPath)
+      .then(() => this.addAboutPage(rootPath))
+      .then(() => this.addFooterToDefaultLayout(rootPath))
+      .then(() => this.addSiteNavToDefaultLayout(rootPath))
+      .then(() => this.addDefaultLayoutToSiteConfig(rootPath))
+      .then(() => this.addTopNavToAddressablePages(rootPath))
```

Comment:
Could we remind user to update their `baseUrl` in `site.json` here?

Reply:
Sorry, I'm not clear on what you mean...the `baseUrl` is set during `markbind init`, right?

Reply:
After `markbind init`, the `baseUrl` in `site.json` is the default `''`. This is unlikely to be the case for most github repositories. I was wondering if it would be good to remind the authors to update it appropriately, or even better, do it for them (maybe we could guess it based on the repository, similar to the navigation)

Reply:
Ah, I see! Perhaps we could set it to be the root directory name?